src/server/src/main/java/io/cassandrareaper/storage/PostgresStorage.java

```diff
@@ -139,26 +136,27 @@ public boolean isStorageConnected() {
   }
 
   @Override
-  public boolean addCluster(Cluster newCluster) throws ReaperException {
+  public boolean addCluster(Cluster newCluster) {
     Cluster result = null;
     try (Handle h = jdbi.open()) {
       String properties = new ObjectMapper().writeValueAsString(newCluster.getProperties());
-      Preconditions.checkState(newCluster.getPartitioner().isPresent(),
-          "Cannot insert cluster with no partitioner.");
-      int rowsAdded
-          = getPostgresStorage(h)
-              .insertCluster(
-                  newCluster.getName(),
-                  newCluster.getPartitioner().get(),
-                  newCluster.getSeedHosts(),
-                  properties);
+      Preconditions.checkState(newCluster.getPartitioner().isPresent(), "Cannot insert cluster with no partitioner.");
+
+      int rowsAdded = getPostgresStorage(h).insertCluster(
```

Comment:
What is going to happen if we try to add back a cluster that was in `DELETED` state?

Reply:
see todo list above: https://github.com/thelastpickle/cassandra-reaper/pull/719#issuecomment-517219637

>  figure out how to re-add (or re-use) deleted clusters,